plugins/auto-sizes/includes/improve-calculate-sizes.php

```diff
@@ -255,15 +259,18 @@ function auto_sizes_get_layout_width( string $alignment ): string {
  * @return string[] The filtered context keys used by the block type.
  */
 function auto_sizes_filter_uses_context( array $uses_context, WP_Block_Type $block_type ): array {
-	// The list of blocks that can consume outer layout context.
-	$consumer_blocks = array(
-		'core/cover',
-		'core/image',
+	// Define block-specific context usage.
+	$block_specific_context = array(
+		'core/cover'   => array( 'max_alignment' ),
+		'core/image'   => array( 'max_alignment', 'container_relative_width' ),
```

Comment:
Why does `core/cover` not receive `container_relative_width`, but `core/image` does?

Reply:
Nice catch! I missed the check for `core/cover` block.

In https://github.com/WordPress/performance/pull/1795/commits/55349c2d8c8d1f091e90e5bc449df3afd3fe8756 i added the context for cover block.